src/node.cc

```diff
@@ -1236,38 +1204,13 @@ void SetupDomainUse(const FunctionCallbackInfo<Value>& args) {
   HandleScope scope(env->isolate());
   Local<Object> process_object = env->process_object();
 
-  Local<String> tick_callback_function_key = env->tick_domain_cb_string();
-  Local<Function> tick_callback_function =
-      process_object->Get(tick_callback_function_key).As<Function>();
-
-  if (!tick_callback_function->IsFunction()) {
-    fprintf(stderr, "process._tickDomainCallback assigned to non-function\n");
-    ABORT();
-  }
-
-  process_object->Set(env->tick_callback_string(), tick_callback_function);
-  env->set_tick_callback_function(tick_callback_function);
-
   CHECK(args[0]->IsArray());
-  env->set_domain_array(args[0].As<Array>());
```

Comment:
I think it was lost in a collapsed comment: we can remove all uses of `domain_array`. Just do a search for `domain_array`.

Reply:
This should be good now. I cleaned up some domain related flags in multiple places ( https://github.com/nodejs/node/pull/16222/commits/821f51dcf1ab72a7bcd71f998cdee9ad5835a192 ) Hop I did not miss any

Reply:
If it compiles without the `domain_array` definition I think it should be good :)